components/Common/Banner/index.module.scss

```diff
@@ -1,4 +1,4 @@
-@import 'styles/_variables';
+@import 'styles/old/_variables';
```

Comment:
We could update this component to not use the old style. And anyway we will have to do it at some point

Reply:
Not the scope of this PR.

Reply:
sure, but should we update that ?

Reply:
Okay, I've added in this PR the changes

Reply:
Oh ok 😁